scripts/bulk_index.pl

```diff
@@ -49,32 +49,40 @@ BEGIN
 
 $indexer->create_index;
 
+my $run_time = time;
 my $loop = IO::Async::Loop->new;
 my $timer = IO::Async::Timer::Periodic->new(
     first_interval => 0,
-    interval       => 15,
-    reschedule     => 'skip',
+    interval       => 20,
+    reschedule     => 'drift',
```

Comment:
crap, this is an unrelated change -- but it results in a much more reliable runtime.